help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,202 @@
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the Salesforce CRM destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the Salesforce CRM destination fields, follow these steps:
+
+1. In the Mapping step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Add new mapping](../../assets/catalog/crm/salesforce/add-new-mapping.png)
+
+1. In the select source field window, when selecting the source field choose the **[!UICONTROL Select attributes]** category and add the mappings desired.
+![Source mapping](../../assets/catalog/crm/salesforce/source-mapping.png)
+
+1. In the select target field window, select the target field and choose the **[!UICONTROL Select identity namespace]** category and add the mappings desired.
+![Target mapping using SalesforceId](../../assets/catalog/crm/salesforce/target-mapping-salesforceid.png)
+
+1. For custom attributes, in the select target field window, select the target field and choose the **[!UICONTROL Select custom attributes]** category, Next provide the desired target attribute name and add the mappings desired.
+![Target mapping using LastName](../../assets/catalog/crm/salesforce/target-mapping-lastname.png)
+
+1. For instance, you could add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
+
+    ||XDM Profile Schema|[!DNL Salesforce CRM] Instance| Mandatory|
+    |---|---|---|---|
+    |Attributes|<ul><li><code>person.name.firstName</code></li><li><code>person.name.lastName</code></li><li><code>personalEmail.address</code></li></ul>|<ul><li><code>FirstName</code></li><li><code>LastName</code></li><li><code>Email</code></li></ul>|
+    |Identities|<ul><li><code>crmID</code></li></ul>|<ul><li><code>SalesforceId</code></li></ul>|Yes|
+
+1. An example using these mappings is shown below:
+![Target mapping LastName](../../assets/catalog/crm/salesforce/mappings.png)
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.html?lang=en#scheduling) step you will need to manually map Platform segments to the custom field attribute in Salesforce.
```

Comment:
I don't think that using bullet points here is appropriate. Generally, bullet points should be used when we are listing something. In this case, I suggest removing all three bullet points from this section.